src/gm.datepickerMultiSelect.js

```diff
@@ -109,6 +111,8 @@ SOFTWARE.
 				return function (scope, element, attrs, ctrls) {
 					link.apply(this, arguments);
 
+					if(!angular.isDefined(scope.$parent.selectedDates)) return;
```

Comment:
I believe that's correct since we just check if property is defined, if attribute exists in directive HTML markup then it means we want multiselect, if it is not defined in markup then we don't proceed with compile and thus we don't define ``selectedDates`` so the rest of code in second compile won't be executed too.